### ISSUE TITLE
Update MacOS jedi-stack documentation

### ISSUE THREAD

**Issue by markjolah** [JCSDA/jedi-stack#165]
Current directions are not in sync with jedi-docs, and lists of homebrew packages are out of order.  It can be difficult for new users to get setup up quickly.  Should we move any of this documentation from jedi-stack to jedi-docs?

**Comment by rickgrubin** [comment]
Agree that the documentation for building on a mac is outdated.

How many of our users are building the stack _from scratch_? My (perhaps uninformed) thoughts are that `jedi-docs` are for users, to understand how to use the tools that are provided.

We could add documentation for developers / building; one could argue persuasively either way (with the tool(s), or in `jedi-docs`). If it moves to `jedi-docs`, to what part -- `docs/jedi-components`, or ?

First thought (best thought?) -- update what's in the `jedi-stack` repository.

**Comment by mmiesch** [comment]
Yes - the rationale behind this was that we do not want to spend a lot of time helping users install netcdf or hdf5 or whatever.  If we put this in the jedi-docs then we open ourselves up to endless questions about building the stack when it doesn't work on their system.  The containers were intended to avoid this.  The jedi-stack was always just intended for experienced developers who know what they are doing and was supposed to be offered "as-is".

So, yes, I do think we should update the instructions but I do not think we should move this to the main jedi documentation.  The jedi-stack should be self-contained.  It's not really part of JEDI and it's a distraction from the main documention.  Those who use HPC modules or containers need not worry about it.  That's what we want to encourage.

**Comment by mmiesch** [comment]
Actually, I think the ideal place to have the Mac jedi-stack instructions would be on a wiki and/or a knowledge base, generated and maintained via the Discourse forums.  That would help keep it thorough and up to date with input from all sorts of people trying on all sorts of systems.
